build.gradle

```diff
@@ -11,7 +11,7 @@ plugins {
 sourceCompatibility = JavaVersion.VERSION_1_8
 
 group 'bio.terra'
-version '0.0.6-SNAPSHOT'
+version '0.0.7a-SNAPSHOT'
```

Comment:
what's the `a` for? I haven't seen  our other repos do this

Reply:
It was just a test version number so we can test it more easily in other repos. Probably a better approach is to publish libraries for testing to `snapshot-local` and those for releases to `release-local`, but we're not setup for that here. I'll bump it to a regular version number `0.0.7` once this gets merged.